Automattic/wp-calypso
# Stats: Updating hover CSS for horizontal bars

#### Proposed Changes

* Adding new hover style for horizontal bars to improve visibility

#### Testing Instructions



* open live branch URL
* navigate to Stats -> Traffic
* hover one of the rows containing horizontal bars and verify that it looks properly (label, value and action icons are visible) across all breakpoints

Before:
![Screenshot 2022-12-22 at 2 25 58 PM](https://user-images.githubusercontent.com/112354940/209034758-665453fe-393e-4272-8ea3-2253b6a9ec4e.png)

After:
![Screenshot 2022-12-22 at 7 31 17 PM](https://user-images.githubusercontent.com/112354940/209071787-2ab758c2-4529-4bbf-8d33-a53d1f40ef1e.png)
![Screenshot 2022-12-22 at 7 30 40 PM](https://user-images.githubusercontent.com/112354940/209071805-497d9b5a-9c7a-4aa8-aa1e-5b1ccbfa43b6.png)



#### Pre-merge Checklist





Related to #70975
